js/components/image.js

```diff
@@ -59,6 +59,11 @@ Fliplet.FormBuilder.field('image', {
       return new Promise(function(resolve, reject) {
         $vm.boundingRect = fileInput.getBoundingClientRect();
 
+        var buttonLabels = ["Take Photo", "Choose Existing Photo", "Cancel"];
+		    if (Modernizr.windows) {
```

Comment:
Wrong indentation

Reply:
wtf? This is what I have here: https://cl.ly/1u1I0d1M2l0V

Reply:
fixed.